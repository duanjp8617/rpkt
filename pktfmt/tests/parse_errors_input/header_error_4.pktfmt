packet FakeProto {
    header = [
        field_1 = Field {
            bit = 8, 
        },
        field_2 = Field {
            bit = 8, 
        },
        field_3 = Field {
            bit = 8388600, 
        },
        field_4 = Field {
            bit = 8, 
        }
    ]
}

