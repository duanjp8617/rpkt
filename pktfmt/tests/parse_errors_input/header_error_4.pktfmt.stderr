ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/header_error_4.pktfmt 2:5-15:5
         ~~~~~~~~~~
 2 |     header = [
 3 |         field_1 = Field {
 4 |             bit = 8, 
......
13 |             bit = 8, 
14 |         }
15 |     ]
     ^^^^^
header error 4:
header byte length is at least 1048577, exceeding the maximum MTU size 1048575
